toolbox/githubctl/main.go

```diff
@@ -204,16 +206,18 @@ func CreateIstioReleaseUploadArtifacts() error {
 			istioRepo, releaseTag, archiveDir); err != nil {
 			return err
 		}
-		if err := u.WriteFile(releaseTagFile, *nextRelease+"\n"); err != nil {
+		if err := u.WriteFile(releaseTagFile, *nextRelease); err != nil {
 			return err
 		}
-		return u.UpdateKeyValueInFile(
-			downloadScript, "ISTIO_VERSION", releaseTag)
+		if *udpateDownloadScript {
+			log.Printf("Updating download script with latest release")
+			return u.UpdateKeyValueInFile(
```

Comment:
why return ?
also probably want to prBody += " and updating download script"

Reply:
he is doing a func inside a func.

Reply:
oic, missed that, thx